projects/retry/fuzz_retry.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import atheris
+import sys
+import random
+with atheris.instrument_imports():
+     from retry import *
+
+def error_method():
+    if random.randint(1,4) == 1:
```

Comment:
We should avoid using `random.randint` because it will make things non-deterministic. If you need an element of randomness here, could you then make it dependent on a global variable and then set this global variable in `TestInput` using some data from the fuzzer?

Reply:
The tricky point of that is this is the retry function. The method will repeat until success where there is no chance to refresh the test input. As a result, it will always return same number and create infinitely loop.

Reply:
Thanks. Changed to use global variable instead.